.rubocop.yml

```diff
@@ -103,35 +88,14 @@ Style/ExpandPathArguments:
 Style/GlobalStdStream:
   Enabled: false
 
-Style/HashEachMethods:
-  Enabled: false
-
 Style/HashLikeCase:
   MinBranchesCount: 4
 
-Style/HashTransformKeys:
-  Enabled: false
-
-Style/HashTransformValues:
-  Enabled: false
-
 Style/ModuleFunction:
   Enabled: false
 
-Style/RedundantFetchBlock:
-  Enabled: false
-
-Style/RedundantFileExtensionInRequire:
-  Enabled: false
-
-Style/RedundantRegexpCharacterClass:
-  Enabled: false
-
 Style/ReturnNil:
   Enabled: true
 
-Style/SlicingWithRange:
```

Comment:
As same as above given the nature of false positives this has I would keep it disabled by default

Reply:
I was getting my info from rubydoc.info which did not mention some of these false positives, I see docs.rubocop.org is better documented (as it should)